Formula/man-db.rb

```diff
@@ -57,6 +58,25 @@ def install
     system "make", "CFLAGS=#{ENV.cflags}"
     system "make", "install"
 
+    # Symlink commands without 'g' prefix into libexec/gnubin and
+    # man pages into libexec/gnuman
+    %w[apropos catman lexgrog man mandb manpath whatis].each do |cmd|
+      (libexec/"gnubin").install_symlink bin/"g#{cmd}" => cmd
+    end
+    %w[accessdb].each do |cmd|
+      (libexec/"gnusbin").install_symlink sbin/"g#{cmd}" => cmd
```

Comment:
1. This is a single file, so no need for 3 lines of code.
2. Same as above: using `gnu` is deceptive. How about `nongnu`?